resources/styles/components/exercises.less

```diff
@@ -130,8 +170,14 @@
       border: 1px solid @tutor-neutral;
       height: 40px;
       margin-right: 0;
+      margin-left: -1px;
+
```

Comment:
Is the `-1px` to remove the border between buttons?  If so, can you just set border-left to `0` for `button+button`